CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ feel free to ask us and community.
 * extend afterLoad() subscriber interface to take LoadEvent ([issue #4185](https://github.com/typeorm/typeorm/issues/4185))
 * relation decorators (e.g. `@OneToMany`) now also accept `string` instead of `typeFunction`, which prevents circular dependency issues in the frontend/browser ([issue #4190](https://github.com/typeorm/typeorm/issues/4190))
 * added support for metadata reflection in typeorm-class-transformer-shim.js ([issue #4219](https://github.com/typeorm/typeorm/issues/4219))
+* added `SelectQueryBuilder#distinct` for building distinct (`SELECT DISTINCT`) queries.
```

Comment:
We could remove this, since we're now using auto generating changelog entries from commit messages.

Reply:
Done. Also added `feat: ` to the commit message so it ought to auto-detect.